app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post, counter_cache: :comments_counter
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  def update_comments_counter
+    post.increment!(:commentsCounter)
+  end
```

Comment:
- [ ] Kindly make sure you fix the above typo error as it will raise an error when trying to run your app. You wrote `post.increment!(:commentsCounter)` instead of  `post.increment!(:comments_counter)`. No column from your `comments` table is named `commentsCounter`